### PR TITLE
Add the story

### PR DESCRIPTION
**Pull request by Qiangong2**
Lots of changes in this one. Story is starting to come together.

### CODE REVIEW

**Comment by Qiangong2** [comment]
@bitcraft 

Have you gotten a chance to look over this?

File: tuxemon/resources/maps/downstairs_test.tmx [hunk 1]
```diff
@@ -78,7 +78,7 @@
   </object>
   <object id="27" name="Go Outside" type="event" x="80" y="112" width="16" height="16">
    <properties>
-    <property name="act1" value="transition_teleport paper_town.tmx,31,7,0.3"/>
+    <property name="act1" value="transition_teleport map1.tmx,44,53,0.3"/>
```
Comment (ShadowApex) [inline tuxemon/resources/maps/downstairs_test.tmx]: This action in the downstairs of the starting house appears to teleport the player to the wrong house. It should teleport the player to 43,46 instead.

File: tuxemon/resources/maps/route1.tmx [hunk 2]
```diff
@@ -117,95 +122,80 @@
...
+    <property name="cond2" value="not npc_exists npc_xerogrund2"/>
    </properties>
   </object>
-  <object id="58" name="ask grunt" type="event" x="224" y="272" width="16" height="16">
+  <object id="69" name="talkin" type="event" x="528" y="272" width="16" height="16">
    <properties>
-    <property name="act1" value="translated_dialog xero_grunt1"/>
-    <property name="act2" value="translated_dialog xero_grunt2"/>
-    <property name="act3" value="translated_dialog xero_grunt3"/>
+    <property name="act1" value="translated_dialog waiting_for_misa"/>
```
Comment (ShadowApex) [inline tuxemon/resources/maps/route1.tmx]: This dialog says to talk to the person to his left, which I think it a little confusing. Technically it is HIS left, but he's to the right of the NPC on the screen. Maybe we should change this dialog to say "Talk to my friend next to me?"

screenshot

Comment (Qiangong2) [reply tuxemon/resources/maps/route1.tmx]: You're right, this should be a quick fix

**Pull request merged**